### PR TITLE
Updated Contribution Process + Re-organized

### PR DESCRIPTION
**Pull request by amsos**
**Description**
Finalize the instructions for the first version of contribution guidelines.  I also re-organized the README structure to look nicer

**Motivation and Context**
Needed to establish the contribution guidelines process

**How Has This Been Written?**
Highly confident

**Types of changes**
_What types of changes does your contribution introduce? Put an `x` in all the boxes that apply:_
- [x] Editing a current section
- [x] Proposing a new section
- [ ] Suggested change to the entire themes of the paper

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,3 +34,25 @@ We host community calls and live editing sessions during each white paper sectio
...
+There are several paths to contribute to the Govrn White Paper.
+1. **Submit an Issue:** Issues are similar to comments in google docs.  They allow you to start a discussion on a certain sections, parts, on entire chapters of the White Paper.  Currently, there are two types of issues to submit, a Comment or a Section Propoosal.
+   - **Comments** should be used when you have a question, idea, or thoughts on an already written part of the paper.  The template for a comment can be found [here](.github/ISSUE_TEMPLATE/comment.md)
+   - **Section Proposals** should be used when you think that a new sub-section or idea should be added to the paper, but aren't yet ready to write it up for a PR (more information on PRs below).  The template for a comment can be found [here](.github/ISSUE_TEMPLATE/section-proposal.md)
+1. **Submit a Pull Request:**  Pull Requests are similar to making or suggesting edits in google docs.  They allow you write or re-write sections of the White Paper.  When you create and submit a pull request, you're making a copy of the current version of the White Paper, writing a new version, and submitting the new version to become the main version for the White Paper.
```
Comment (christinevandev) [inline README.md]: allow you to write

**Review by christinevandev — CHANGES_REQUESTED** [review]
Line 47 - add "to"

File: README.md (hunk 1 above)
Comment (amsos) [reply README.md]: fixed

**Review by amsos — COMMENTED** [review]
Finished

**Comment by amsos** [comment]
@christinevandev line 47 fixed for your review

**Comment by christinevandev** [comment]
Fixed typo

**Pull request merged**